README.md

```diff
@@ -108,6 +109,15 @@ We, at Public Lab use [openssl](https://github.com/ruby/openssl) gem to provide
 You can also use http (unsecure connection) on the port number 3000 by going to 'http://localhost:3000'. We use port number 3001 for 'https' and port number 3000 for 'http' connection.
 Secure connection is needed for OAuth authentication etc.
 
+## Login
+
+Once you complete the installation, use any of these credentials to login in to the PL website in your local development / testing environment to gain additional permissions for only logged in users. Each one comes with its own set of permissions, but besides that the experience across them is pretty much the same.
+
+**login**: `admin`, `moderator`, or `user` 
```

Comment:
```suggestion
**username**: `admin`, `moderator`, or `user` 
```

login is fine but any chance it can confuse a first-timer?

Reply:
Also, want to highlight, we mention about username and password in Step 10 of installation too - https://github.com/publiclab/plots2/pull/5544/files#diff-04c6e90faac2675aa89e2176d2eec7d8R100

Reply:
@gauravano ah I see! I didn't notice that before. I think having both is fine. because in the checklist it may be overlooked and I also provide a little explanation as to why we have 3 different ones.